README.md

```diff
@@ -29,26 +29,103 @@ git clone https://github.com/sap/service-fabrik-boshrelease
 cd service-fabrik-boshrelease
 ```
 
+### Deploying Service-Fabrik
 
-### Deploy from Sources
+#### Creating and uploading release:
 
-If you need to modify the sources (beyond the example manifest):
+  ##### Deploying from Sources
+
+  If you need to modify the sources (beyond the example manifest):
+  ```shell
+  ./scripts/update
+  bosh create release --force && bosh upload release # repeat these commands on every change
+  ```
+
+  #### Deploying from Release
+
+  If you do not need to modify the sources (beyond the example manifest):
+  ```shell
+  bosh upload release $(ls -1rv releases/service-fabrik/service-fabrik-*.yml | head -1)
+  ```
+
+#### Uploading the cloud-config (for AWS):
```

Comment:
Put the boshlite one before, I dont think we need two sections, have one section as Uploading the cloud-config, and mention that for AWS, var files need to be updated.

Reply:
ok.. will modify this..